activities/procurement/process.md

```diff
@@ -9,4 +9,6 @@ This is how we procure all services and some goods.
 1. We request a proposal or quote.
 2. We check if price and value are in line with the market.
 3. We check that the proposed procurement meets our [procurement principles](principles.md).
-4. The [operations team](../../organization/staff.md#operations) places the order, after approval by the chief executive and board of directors, if needed.
+4. The [operations team](../../organization/staff.md#operations) places the order, or approves the proposal after sign-off by the chief executive and board of directors, if needed.
+5. When we procure services of a contractor, we draft a service agreement based on the approved proposal. Both parties need to sign the agreement and ensure adherence to the terms.
+6. Depending on the duration and scope of the assignment, we also expect time sheets with each invoice. The payment will only be processed if the time sheet and invoice are in order.
```

Comment:
Very good, I'd add checking the deliverables on payment of the invoice. Alternatively the payment of invoices could be a separate process as it is likely to be done at a different moment. You could also do that in a separate pull request.

Reply:
Added that deliverables will be reviewed.